src/webrtc/call.ts

```diff
@@ -2024,7 +2024,7 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
 
                 this.sendVoipEvent(EventType.CallNegotiate, {
                     lifetime: CALL_TIMEOUT_MS,
-                    description: this.peerConn!.localDescription?.toJSON(),
+                    description: this.peerConn!.localDescription?.toJSON() as RTCSessionDescription,
```

Comment:
This possibly casts `null`; is that fine?  Should there be a null-assertion operator on `localDescription`?  That's done elsewhere in this file.

Also, with the Typescript update, `localDescription` is the same type as `description` even without casting its JSON form.  Would it be safe to use the non-JSON format here?

Reply:
About the `RTCSessionDescription` casting, I don't know if it still working if we are using the non-JSON format.
Because, normally it should be the non-json format here. There is an error but I don't want to create any regression in an TS update.